### ISSUE TITLE
Installer stuck half way after reinstall

### ISSUE THREAD

**Issue by NomadMTG** [Manuel-777/MTG-Arena-Tool#636]
Hi, I tried to reinstall MTG Arena Tool by deleting all the folders it appeared in Windows Search (in App Data and Program Files) and then running the .exe for Windows again.

The installer then froze halfway and even half an hour later didn't move. I tried downloading different versions (2.11.0 and 2.11.3) but run into the same problem with all of them. I've attached a screenshot of what it looks like.

I can't find Arena Tool in the Add/Remove programs section and I'm not sure if there's some other area I might need to delete files from (like the registry?).

![image](https://user-images.githubusercontent.com/56621926/66889472-4e0ac000-f02e-11e9-9626-81b80e1e5e47.png)

**Comment by Manuel-777** [comment]
Never seen that error myself, but have you tried running the installer with Admin rights ?

**Comment by NomadMTG** [comment]
> Never seen that error myself, but have you tried running the installer with Admin rights ?

Just tried, same result I'm afraid!

**Comment by NomadMTG** [comment]
I think I solved it!

I went back to Add/Remove Programs and MTG Arena Tool was listed there (although I don't remember seeing it before). When I tried to remove it I got the message that the files were already missing, would I like to delete the shortcut. I said yes and after that the installer worked!

Hopefully this can help some other people if they have the same problem. Thanks for the great program!
